docs/inputs/toml.rst

```diff
@@ -442,12 +442,12 @@ Sectors contain a number of subsections:
         are registered via :py:func:`~muse.investments.register_investment`. At time of
         writing, three are available:
 
+        - "scipy" solver (default): Formulates investment as a true LP problem and solves it using
+          the `scipy solver`_.
+
         - an "adhoc" solver: Simple in-house solver that ranks the technologies
           according to cost and service the demand incrementally.
 
-        - "scipy" solver: Formulates investment as a true LP problem and solves it using
-          the `scipy solver`_.
-
         - "cvxopt" solver: Formulates investment as a true LP problem and solves it
```

Comment:
Is this actually working? I've never tried it or had to deal with it.

Reply:
A quick test suggests the answer is no (at least in the current MUSE version - it's not covered by the tests so might have been accidentally broken along the way)